### DISCUSSION TITLE
How change lucid model table name from outside

### DISCUSSION THREAD

**Discussion by AhmadPirzargar** [adonisjs/core#1347] [Help]
We use adonis-ACL. We want to change "Role" and "Permission" table names to be consistent with our naming conventions. Is there any way to set table names as desired?

thank you

**Comment by alainrc2005** [comment]
Inside the model

static get table () {
        return 'roles_permission' // real database table name
      }

**Reply by AhmadPirzargar** [reply]
Thank you for your response but models in [adonis ACL](https://www.npmjs.com/package/adonis-acl) in the package and cannot directly access to models (role, permission)

**Reply by thetutlage** [reply]
You must open the issue with the package author. It is not maintained by the core team

**Reply by AhmadPirzargar** [reply]
Thanks a lot for your response,
Is there any way to use the life cycle hook for this purpose؟
